apps/builder/app/shared/nano-states/props.ts

```diff
@@ -718,6 +718,11 @@ export const subscribeResources = () => {
         cacheByKeys.set(cacheKey, undefined);
       }
 
+      const missingValues = Array.from(missing.values());
+      if (missingValues.length === 0) {
+        return;
+      }
```

Comment:
does it invalidate resource when click "refresh" button?

Reply:
From logic I see if missing is empty, nothing is loaded